ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -210,6 +210,8 @@
 {% include 'eos/static-routes.j2' %}
 {# IPv6 Static Route #}
 {% include 'eos/ipv6-static-routes.j2' %}
+{# IP NAT #}
+{% include 'eos/ip-nat.j2' %}
```

Comment:
So without asking you to change anything at this stage, I just realized looking at my test device that the nat config is not in one single place in the running config which means AVD won't be rendering the config in the correct order (which we "_try_" to do as much as possible).

```
ip nat translation max-entries 100000
ip nat translation low-mark 50
ip nat translation max-entries 1000 host
ip nat translation low-mark 50 host
ip nat translation max-entries 200 10.0.0.2
ip nat translation max-entries 100 10.0.0.1
ip nat kernel buffer size 64
!
hostname Host1
!
spanning-tree mode mstp
!
vrf instance mgmt
!
aaa authorization exec default local
!
interface Ethernet1
!
interface Ethernet2
!
interface Ethernet3
!
interface Ethernet4
!
interface Ethernet5
!
interface Ethernet6
!
interface Ethernet7
!
interface Ethernet8
!
interface Management1
!
ip routing
no ip routing vrf mgmt
!
ip route vrf mgmt 0.0.0.0/0 1.1.1.1
!
ip nat pool prefix_16 prefix-length 16
   range 10.0.0.1 10.0.255.254
   range 10.1.0.0 10.1.255.255 1024 65535
   utilization threshold 1 action log
!
ip nat pool prefix_24 prefix-length 24
   range 10.3.0.1 10.3.0.254
   range 10.3.1.0 10.3.1.255 1024 65535
!
ip nat pool prefix_32 prefix-length 32
   range 10.2.0.1 10.2.0.1
!
ip nat synchronization
   description test sync config
   expiry-interval 60
```

an alternative could be to use two templates for the config (keeping one for the documentation) as it is quite straighforward to identify what goes where.

Thoughts @kornoa / @ClausHolbechArista  ?

Reply:
I also considered splitting the NAT config. It is just the question whether the simplicity in Jinja or the order in the EOS config prevails. I will simply follow your guidelines here.

Reply:
Copy.

Reply:
Fixed in https://github.com/aristanetworks/ansible-avd/pull/2747/commits/dbb7ed0fbc95f0b52259dcef05d26c31291b581c